climada_petals/hazard/tc_tracks_forecast.py

```diff
@@ -529,3 +539,110 @@ def _check_variable(var, n_ens, varname=None):
 
         else:
             raise ValueError
+
+    @classmethod
+    def read_cxml(cls, cxml_path: str, xsl_path: str = None):
+        """Reads a cxml (cyclone xml) file and returns a class instance."""
+        df = cls._cxml_to_df(cxml_path=cxml_path, xsl_path=xsl_path)
+        df_groupby = df.groupby(
+            ["disturbance_no", "baseTime", "basin", "cycloneNumber", "member"],
+            sort=False,
+            dropna=False,
+        )
+        instance = cls()
+        instance.data = [cls._fcastdf_to_ds(subdf) for _, subdf in df_groupby]
+
+        return instance
+
+    @staticmethod
+    def _cxml_to_df(
+        cxml_path: str, xsl_path: str = None, basin_env_pressures: dict = None
+    ):
+        """Read a cxml v1.1 file; may not work on newer specs."""
+        try:
+            import lxml.etree as et
+            import io
+        except ModuleNotFoundError:
+            LOGGER.exception("Please install the lxml module manually")
```

Comment:
1) This just logs the message but keeps on going, bound to fail later on when et is used.
2) Judging from the code, it is not clear which module wasn't found.
```suggestion
        except ModuleNotFoundError as missing_module:
            LOGGER.exception("%s, Please install it manually", str(missing_module))
```